spec.html

```diff
@@ -35918,16 +35690,28 @@ <h1>Properties of Set Instances</h1>
       <h1>Set Iterator Objects</h1>
       <p>A Set Iterator is an ordinary object, with the structure defined below, that represents a specific iteration over some specific Set instance object. There is not a named constructor for Set Iterator objects. Instead, set iterator objects are created by calling certain methods of Set instance objects.</p>
 
-      <emu-clause id="sec-createsetiterator" aoid="CreateSetIterator">
+      <emu-clause id="sec-createsetiterator" aoid="CreateSetIterator" oldids="sec-properties-of-set-iterator-instances,table-51,table-internal-slots-of-set-iterator-instances">
         <h1>CreateSetIterator ( _set_, _kind_ )</h1>
         <p>The abstract operation CreateSetIterator takes arguments _set_ and _kind_. This operation is used to create iterator objects for Set methods that return such iterators. It performs the following steps when called:</p>
         <emu-alg>
           1. Perform ? RequireInternalSlot(_set_, [[SetData]]).
-          1. Let _iterator_ be ! OrdinaryObjectCreate(%SetIteratorPrototype%, &laquo; [[IteratedSet]], [[SetNextIndex]], [[SetIterationKind]] &raquo;).
-          1. Set _iterator_.[[IteratedSet]] to _set_.
-          1. Set _iterator_.[[SetNextIndex]] to 0.
-          1. Set _iterator_.[[SetIterationKind]] to _kind_.
-          1. Return _iterator_.
+          1. Let _closure_ be a new Abstract Closure with no parameters that captures _set_ and _kind_ and performs the following steps when called:
+            1. Let _index_ be 0.
+            1. Let _entries_ be the List that is _set_.[[SetData]].
+            1. NOTE: the number of elements of _entries_ mentioned below must be redetermined every iteration.
```

Comment:
Same as above. If we're explicitly setting this value in a step, there's no need to clarify anything in a note.